packages/core/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @wagmi/core
 
+## 0.5.5
+
+### Patch Changes
+
+- [#912](https://github.com/wagmi-dev/wagmi/pull/912) [`e529e12`](https://github.com/wagmi-dev/wagmi/commit/e529e125c713ed3ef24a59c6bf226fe4deee7ac9) Thanks [@zouhangwithsweet](https://github.com/zouhangwithsweet)! - Added BitKeep and Math Wallet to injected flags
```

Comment:
Separate out Math Wallet before merging in (PR https://github.com/wagmi-dev/wagmi/pull/910 didn't have changeset)